src/modules/pages/components/RouteLayouts/UserNavigation.css

```diff
@@ -142,3 +142,20 @@
   color: var(--colony-white);
   box-shadow: 0px 2px 11px var(--drop-shadow);
 }
+
+.reputation {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-right: 20px;
+  padding: 0 5px;
+  height: 26px;
+  border-radius: var(--radius-normal);
+  background-color: color-mod(var(--temp-grey-blue-7) alpha(10%));
+}
+
+.reputation span {
+  display: inline-block;
+  vertical-align: middle;
+  font-size: 9px; /* it seems we start useing this font regulary, time for var? */
+}
```

Comment:
So my only complaint about this is here the size of the text. 

It's too small:

![Screenshot from 2021-03-15 21-41-07](https://user-images.githubusercontent.com/1193222/111212271-149a0380-85d8-11eb-8ed6-9b25c7fb0e00.png)

I know this is not your fault, as the specs show it like that:

![Screenshot from 2021-03-15 21-46-49](https://user-images.githubusercontent.com/1193222/111212325-1fed2f00-85d8-11eb-9e60-b644e138efdf.png)

I think that the specs are wrong. I think the font should be a bigger size and the side padding should be bigger. 

If you really want to, you can ask Karol, but these change should be done for consistency, otherwise the user navigation will look bad.

Here's how I changed them locally:

![Screenshot from 2021-03-15 21-50-19](https://user-images.githubusercontent.com/1193222/111212821-c3d6da80-85d8-11eb-99e5-9cb821af9c8f.png)

Increased the font size to 13px and removed the vertical-align _(it aligns perfectly with a font of 13px)_, and increased the side padding to 7px

```js
.reputation {
  ...
  padding: 0 7px;
  ...
}

.reputation span {
  display: inline-block;
  font-size: var(--size-smallish);
}
```

Reply:
i love it, i will make this change, and i also think that is something wrong with specs, because it looks like things are similar size, but when you apply those styles it looks a lot different